src/components/Navbar/Navbar.module.scss

```diff
@@ -110,63 +123,80 @@
   border: 0px;
 }
 
+.AnimationButton * {
+  color: black;
```

Comment:
zmień na zmienną tylko z theme pliku

Reply:
to wywalić w ogóle trzeba testowałam czy w ogóle kolory tak zmieni (nie zmianiało)